### PR TITLE
Restrict facet values by prefixes and/or matches on a per field basis…

### PR DESCRIPTION
**Pull request by hajoseng**
…. This is done by setting the arrays facet_prefix_by_field or facet_matches_by_field within the Results_Settings part of facets.ini

### CODE REVIEW

File: config/vufind/facets.ini [hunk 1]
```diff
@@ -134,6 +134,12 @@ facet_limit = 30
 ; Override facet_limit on a per-field basis using this array:
 ;facet_limit_by_field[format] = 50
 
+; Prefix a facet on a per-field basis:
```
Comment (demiankatz) [inline config/vufind/facets.ini:137]: Might help to expand this comment to something like "Limit facets based on a prefix on a per-field basis:"

File: config/vufind/facets.ini [hunk 2]
```diff
@@ -134,6 +134,12 @@ facet_limit = 30
...
+;facet_prefix_by_field[building] = 22
+
+; Select facet value by a regualra expression on a per-field basis:
```
[the remark this answers is not in the document]
Comment (demiankatz) [reply config/vufind/facets.ini:140]: Might also help to clarify to something like "Filter facet values to those matching a regular expression on a per-field basis."

[the remark this answers is not in the document]
Comment (hajoseng) [reply config/vufind/facets.ini]: Thank you very much. I've included your suggestions into the comments.

**Comment by demiankatz** [comment]
Thank you! It's working well for me, so I'm merging now.

**Pull request merged**